server/apps/immich/src/api-v1/asset/asset.controller.ts

```diff
@@ -131,25 +140,29 @@ export class AssetController {
     @Param('assetId') assetId: string,
     @Query(new ValidationPipe({ transform: true })) query: GetAssetThumbnailDto,
   ): Promise<any> {
-    await this.assetService.checkAssetsAccess(authUser, [assetId]);
+    await this.assetService.checkAssetsAccess(authUser, [assetId], false, query.key);
```

Comment:
We should be able to accomplish this with `authUser` alone, no need to pass in extra query params.